packages/mermaid/src/diagrams/sequence/sequenceDiagram.spec.js

```diff
@@ -173,13 +173,12 @@ describe('more than one sequence diagram', () => {
 describe('when parsing a sequenceDiagram', function () {
   beforeEach(function () {
     // diagram.db = sequenceDb;
-    // diagram.db.clear();
+    //
```

Comment:
May be, we do not need those comments at all

Reply:
Removed